dask/dataframe/groupby.py

```diff
@@ -1866,30 +1859,28 @@ def prod(
 
     @derived_from(pd.core.groupby.GroupBy)
     def min(self, split_every=None, split_out=1, shuffle=None, numeric_only=no_default):
-        if numeric_only is no_default:
-            numeric_only = False
```

Comment:
Note to readers that `numeric_only=` has always defaulted to `False` here (not always the case with other methods)